lib/recurly/payment-methods/gateways/base.js

```diff
@@ -0,0 +1,41 @@
+import Emitter from 'component-emitter';
+
+const notImplementedError = new Error('not implemented');
+
+class Base extends Emitter {
```

Comment:
Hmmm. I'm not fully understanding the need for this class instead of just extending from Emitter?

Reply:
This is just an abstract class to define what methods need to be implemented in order to add a new PaymentMethod gateway strategy.
For stripe, we just need to implement these methods.
We can rip off this class but will be really hard to know what needs to be done to support Stripe Payment Methods.